### PR TITLE
Turn on canceled appts job in queue

### PR DESCRIPTION
**Pull request by TMcMeans**
Cancelation emails are generating on local, but still need to test in staging.

### CODE REVIEW

File: lib/queue.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const bull = (name, url = REDIS_URL, opts) => {
 
 export const UIQueues = [
   bull('fetch/apptsNewInLastMin'),
-  // bull('fetch/apptsUpdatedInLastMin'),
+  bull('fetch/apptsUpdatedInLastMin'),
```
Comment (craigspaeth) [inline lib/queue.js]: We don't want this back in right? Just the line below it

Comment (TMcMeans) [reply lib/queue.js]: Oh woops! Editing right now.

Comment (TMcMeans) [reply lib/queue.js]: Thank god for code review :)

**Comment by craigspaeth** [comment]
LGTM! Thanks!

**Pull request merged**